src/metrics/commits-per-pr/index.ts

```diff
@@ -10,9 +8,11 @@ import { Commits, CommitsPerPrOutput, CommitsPerPrPayload } from "./interfaces";
 import { getTimestamp } from "../../shared/get-timestamp";
 import { LogErrors } from "../../shared/log-messages";
 import { logger } from "../../core";
+import { Octokit } from "@octokit/rest";
 
 export const collectCommitsPerPrMetrics = async (
-  triggerEvent: SignedTriggerEvent
+  triggerEvent: SignedTriggerEvent,
+  repoClient: Octokit
 ): Promise<MetricData<MetricSignature.CommitsPerPr>> => {
   const payload = triggerEvent.payload as CommitsPerPrPayload;
 
```

Comment:
to avoid unused var in some metrics

we would need to check here if `repoClient` is set and if not do an early exit

Reply:
Will throw an exception then.